eng/Versions.props

```
@@ -25,23 +25,23 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.149-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.6.0</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>17.7.38-preview</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>17.7.124-preview</VisualStudioEditorPackagesVersion>
     <!-- This should generally be set to $(VisualStudioEditorPackagesVersion),
          but sometimes EditorFeatures.Cocoa specifically requires a newer editor build. -->
     <VisualStudioMacEditorPackagesVersion>$(VisualStudioEditorPackagesVersion)</VisualStudioMacEditorPackagesVersion>
```

Until the VS Editor CI issue is fixed, this should be sufficient to unblock you, I think:

```suggestion
    <VisualStudioMacEditorPackagesVersion>17.7.38-preview</VisualStudioMacEditorPackagesVersion>
```